src/java/textdisplay/Folio.java

```diff
@@ -345,6 +345,33 @@ public static int createFolioRecord(String collection, String pageName, String i
          //j.close();
       }
    }
+    public static boolean exists(int folioID) throws SQLException{
+       boolean result = false;
+       String query = "select * from folios where pageNumber=?";
+       Connection j = null;
+       PreparedStatement ps = null;
+      try {
+         j = getConnection();
+         ps = j.prepareStatement(query);
+         ps.setInt(1,folioID);
+         //ps.setString(1, canvas);
+         //
+         //
+         ResultSet rs = ps.executeQuery();
+         if (rs.next()) {
+            int ans = rs.getInt(11);
+            if(ans>0){
+                return true;
+            }
+
```

Comment:
Cleanup this whitespace, and any others you notice.

--Note that you can use a [linter](https://lightrun.com/java/top-10-java-linters/) before submitting code, which will 'prettify' it and take care of this kind of stuff automatically.  You should be able to find one you can plug into Netbeans.